### PR TITLE
Chore/import http client module implicit

### PR DESCRIPTION
**Pull request by NachoVazquez**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

When using the `LumberjackHttpDriver` is necessary to import the `HttpClientModule`

Issue Number: N/A

## What is the new behavior?

The `LumberjackHttpDriverModuleRoot` implicitly imports the `HttpClientModule`. 

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: libs/ngworker/lumberjack/http-driver/src/lib/configuration/lumberjack-http-driver.module.spec.ts [hunk 1]
```diff
@@ -57,7 +56,6 @@ const createHttpDriver = (
 ) => {
   TestBed.configureTestingModule({
     imports: [
-      HttpClientTestingModule,
```
Comment (LayZeeDK) [inline libs/ngworker/lumberjack/http-driver/src/lib/configuration/lumberjack-http-driver.module.spec.ts:60]: We must still include `HttpClientTestingModule`, but import it after other modules.

File: libs/ngworker/lumberjack/http-driver/src/lib/configuration/lumberjack-http-driver.module.spec.ts [hunk 2]
```diff
@@ -79,7 +77,6 @@ const createHttpDriverWithOptions = (
...
```
Comment (LayZeeDK) [inline libs/ngworker/lumberjack/http-driver/src/lib/configuration/lumberjack-http-driver.module.spec.ts:82]: We must still include `HttpClientTestingModule`, but import it after other modules.

**Review by LayZeeDK — CHANGES_REQUESTED** [review]

File: libs/ngworker/lumberjack/http-driver/src/lib/configuration/lumberjack-http-driver.module.spec.ts (hunk 1 above)
Comment (NachoVazquez) [reply libs/ngworker/lumberjack/http-driver/src/lib/configuration/lumberjack-http-driver.module.spec.ts:60]: It is not needed here really, we don't perform any HTTP call.

File: libs/ngworker/lumberjack/http-driver/src/lib/configuration/lumberjack-http-driver.module.spec.ts (hunk 2 above)
Comment (NachoVazquez) [reply libs/ngworker/lumberjack/http-driver/src/lib/configuration/lumberjack-http-driver.module.spec.ts:82]: Not necessary in here.

**Review by LayZeeDK — APPROVED** [review]

**Pull request merged**